src/test/java/org/springframework/cloud/deployer/spi/kubernetes/DeploymentPropertiesResolverTests.java

```diff
@@ -37,7 +37,7 @@ public void testRestartPolicy(boolean isDeprecated) {
 		DeploymentPropertiesResolver deploymentPropertiesResolver = getDeploymentPropertiesResolver(isDeprecated, kubernetesDeployerProperties);
 		Map<String, String> properties = new HashMap<>();
 		RestartPolicy restartPolicy = deploymentPropertiesResolver.getRestartPolicy(properties);
-		Assertions.assertThat(restartPolicy).isEqualTo(RestartPolicy.Always);
+		Assertions.assertThat(restartPolicy).isEqualTo(RestartPolicy.Never);
```

Comment:
Currently, with the scheduler properties deprecation we have got the RestartPolicy set to `Never` by [default](https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/blob/main/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java#L563). @cppwfs @onobc Could you double check if this doesn't have any impact with our Stream apps from the SCDF side?